src/ai/ai_gui.cpp

```diff
@@ -14,6 +14,7 @@
 #include "../querystring_gui.h"
 #include "../stringfilter_type.h"
 #include "../company_base.h"
+#include "../company_func.h"
```

Comment:
Could be extern _local_company instead of an #include ?

Reply:
Yeah should work too.